examples/basics/pages/en/users.js

```diff
@@ -14,6 +14,9 @@ const siteConfig = require(process.cwd() + '/siteConfig.js');
 
 class Users extends React.Component {
   render() {
+    if ((siteConfig.users || []).length === 0) {
```

Comment:
We don't use this style of "exists" checking too much, but I am cool with it. I think we use it in [one other place](https://github.com/facebook/Docusaurus/blob/master/lib/core/Site.js#L95).